### ISSUE TITLE
Make the division sub-plots (of the sequences over time plot) sharable

### ISSUE THREAD

**Issue by chaoran-chen** [GenSpectrum/cov-spectrum-website#183]
The division sub-plots are currently not sharable because the sample selector (and the API) do not allow filtering by divisions. The (current/initial) implementation filters the data but passes the original selector.

https://github.com/cevo-public/cov-spectrum-website/commit/8a86592d2b396ea452b64a2036fd3a92253d210f#diff-794371d78f8ad734a009671ae49012f758b400899b3a79c4c44790bcea18ba4cR73-R83

---

In other words: Bring back the "Export" button!

![image](https://user-images.githubusercontent.com/18666552/123660548-0cd33780-d834-11eb-8c49-2327446bc0c9.png)

**Comment by spacephoton** [comment]
Maybe better not to call it "region 1", might be confusing when shared seperately

**Comment by spacephoton** [comment]
how did you come up with these region?

**Comment by chaoran-chen** [comment]
Ah, I should certainly explain it somewhere.. These are the regions by the Sentinella influenza monitoring system.

The numbering is common and used for example in:

- https://www.bag.admin.ch/bag/de/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/saisonale-grippe---lagebericht-schweiz.html
- https://covariants.org/per-country
- https://github.com/CovSeqCH/SequencingStats_and_Variants
